examples/vision/mixup.py

```diff
@@ -30,18 +30,27 @@
 transforms for a given dataset, medical imaging datasets, for example. mixup can be
 extended to a variety of data modalities such as computer vision, naturallanguage
 processing, speech, and so on.
-
-This example requires TensorFlow 2.4 or higher.
 """
 
 """
 ## Setup
 """
 
+import os
+
+os.environ["KERAS_BACKEND"] = "tensorflow"
```

Comment:
Is the example TF-specific or can it run with another backend?

Reply:
Yes, this example is TF-only